src/components/DateRangePicker.jsx

```diff
@@ -281,6 +282,17 @@ class DateRangePicker extends React.PureComponent {
     this.container = ref;
   }
 
+  closeDayPicker() {
+    const { onFocusChange } = this.props;
+    this.setState({
+      isDateRangePickerInputFocused: false,
+      isDayPickerFocused: false,
+      showKeyboardShortcuts: false,
+    });
+
+    onFocusChange(null);
```

Comment:
should this maybe be in a callback to setState, so it's sure to happen after the state is set?

Reply:
Sure I can do that.